c7n/resources/cw.py

```diff
@@ -166,6 +166,96 @@ def get_arns(self, resources):
         return [r['arn'][:-2] for r in resources]
 
 
+@resources.register('insight-rule')
+class InsightRule(QueryResourceManager):
+
+    class resource_type(TypeInfo):
+        service = 'cloudwatch'
+        arn_type = 'insight-rule'
+        enum_spec = ('describe_insight_rules', 'InsightRules', None)
+        name = id = 'Name'
+        universal_taggable = True
```

Comment:
Should this be changed to `object()` since we don't have to maintain any compatibility